az-core/src/main/java/azkaban/utils/PropsUtils.java

```diff
@@ -93,6 +98,45 @@ public static Props loadProps(final Props parent, final File... propFiles) {
     }
   }
 
+  /**
+   * Load plugin properties
+   * @param pluginDir  plugin's Base Directory
+   * @param pluginType plugin type (viewer, trigger, alter)
+   * @return The properties
+   */
+  public static Props loadPluginProps(final File pluginDir, String pluginType) {
```

Comment:
not clear to me the purpose of this method, also pluginType doesn't seem to have point here. Let's talk offline.

Reply:
yeah. Let's do it.

Reply:
addressed the concern offline.